### PR TITLE
Using latest version of FastTwitch :sparkles:

### PR DESCRIPTION
**Pull request by dtaylor-us**

### CODE REVIEW

**Comment by outcastgeek** [comment]
Could you collapse and provide a better message for the commits ?

File: src/main/endpoints.cljs [hunk 1]
```diff
@@ -8,11 +8,18 @@
...
-            [util.os :as os]
-            [express.web-api :as web]
+            [bidi.bidi :refer [path-for]]
+            [fast-twitch.nav :refer [cached-routes]]
+            [fast-twitch.os :as os]
+            [fast-twitch.web-api :as web]
             [ui.components.core :as cmpt]
             [services.core :as svc]))
 
+(defn redirect [route-key]
```
Comment (outcastgeek) [inline src/main/endpoints.cljs]: Isn't already defined in FastTwitch?

Comment (dtaylor-us) [reply src/main/endpoints.cljs]: Yea it is broken so I was using this until you merge pull requet

**Comment by outcastgeek** [comment]
Thx

**Pull request merged**